packages/esm-patient-common-lib/src/dashboards/dashboard-extension.scss

```diff
@@ -0,0 +1,8 @@
+.menu {
+  display: flex;
+  align-items: center;
+}
+
+.icon {
+  margin-right: 0.5rem;
```

Comment:
Please use the [Carbon spacing tokens](https://carbondesignsystem.com/elements/spacing/overview/) for things like this (`use @carbon/layout; margin: layout.$spacing-03;`